starknet_devnet/accounts.py

```diff
@@ -42,11 +46,9 @@ def add(self, account):
     def __generate(self):
         """Generates accounts without deploying them"""
         random_generator = random.Random()
-        self.__initial_balance = self.__initial_balance
+        random_generator.seed(self.__seed)
 
-        random_generator.seed(
-            self.__seed if self.__seed is not None else random_generator.getrandbits(32)
-        )
+        self.__initial_balance = self.__initial_balance
```

Comment:
Looks suspicious to me.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/281/commits/7685c2c09a14763cfd2cdfd17637806040c122bd